components/brave_rewards/core/legacy/media/github.cc

```diff
@@ -212,32 +209,35 @@ void GitHub::ProcessActivityFromUrl(uint64_t window_id,
   }
 
   engine_->database()->GetMediaPublisherInfo(
-      media_key, std::bind(&GitHub::OnMediaPublisherActivity, this, _1, _2,
-                           window_id, visit_data, media_key));
+      media_key, base::BindOnce(&GitHub::OnMediaPublisherActivity,
+                                weak_factory_.GetWeakPtr(), window_id,
+                                visit_data, media_key));
 }
 
 void GitHub::ProcessMedia(const base::flat_map<std::string, std::string> parts,
                           const mojom::VisitData& visit_data) {
   const std::string user_name = GetUserNameFromURL(visit_data.path);
   const std::string url = GetProfileAPIURL(user_name);
   auto iter = parts.find("duration");
-  uint64_t duration = iter != parts.end() ? std::stoull(iter->second) : 0U;
+  uint64_t duration = 0;
+  if (iter != parts.end()) {
+    base::StringToUint64(iter->second, &duration);
```

Comment:
I think we should check the return value here — there's a few other instances as well.

Reply:
Unfortunately, it's not really clear at some of the call sites what the failure semantics should be. `base::ToString*` will do a best effort conversion, which is probably the safest way to go in this PR.